src/handle_swap_sign_transaction.c

```diff
@@ -10,15 +10,31 @@
 #include "nbgl_use_case.h"
 #endif  // HAVE_NBGL
 
+// Remember if we have been started by the Exchange application or not
+bool G_called_from_swap;
```

Comment:
Already declared in main.c, no?

Reply:
Ah ok, I saw you removed it from main...

I am ok, for dure, but just don't know why??

Reply:
I think the amount / scope of globals is a but much and a bit of a mess..
Long term goal is to move each global in its module